libs/ledger-state/src/Cardano/Ledger/State/Query.hs

```diff
@@ -530,7 +531,8 @@ getDStateNoSharing dstateId = do
       rws <- selectList [RewardDstateId ==. dstateId] []
       forM rws $ \(Entity _ Reward {..}) -> do
         Credential credential <- getJust rewardCredentialId
-        pure (Keys.coerceKeyRole credential, rewardCoin)
+        pure (Keys.coerceKeyRole credential, UM.RDPair (UM.compactCoinOrError rewardCoin) (UM.CompactCoin 0))
+  -- FIXME the deposit is not accounted for ^
```

Comment:
I realize that you did not introduce this, but I'm realizing now that I do not understand the original FIXME. is this tool not able to properly load the deposits into the database?

Reply:
Yes, anything that changes the data structures breaks this tool. Alexei promised to fix this. I will remind him. I added this comment.

 -- FIXME the deposit is not accounted for ^
  -- The PR ts-keydeposit-intoUMap breaks this tool since it changes the DPState data type.
  -- https://github.com/input-output-hk/cardano-ledger/pull/3217
  -- All the FIXME-s in this file will have to be addressed in a follow on PR

And Alexey agreed to fix it later.